lib/widget/evaluation_form/post-op-home/abnormal_symptom_form.dart

```diff
@@ -0,0 +1,256 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'post-op-home_page.dart';
+
+class AbnormalSymptomForm extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _AbnormalSymptomFormState();
+  }
+}
+
+class _AbnormalSymptomFormState extends State<AbnormalSymptomForm> {
+  var _value = false;
+  var _value2 = false;
+  var _value3 = false;
+  var _value4 = false;
+  var _value5 = false;
```

Comment:
Should it be Array List of sth? instead of 5 separated variable?

Reply:
As well as the other forms.

Reply:
It use for the set state when changing the answer. And I don't know how to handle it when I set state when keep it as an array list?